JeMPI_Apps/JeMPI_BackupRestoreAPI/src/main/java/org/jembi/jempi/backuprestoreapi/JsonFieldsConfig.java

```diff
@@ -0,0 +1,90 @@
+package org.jembi.jempi.backuprestoreapi;
```

Comment:
@sushantpatil1214 not sure we need this file

Reply:
Its packge defination.